CourseGrab/LoginViewController.swift

```diff
@@ -2,21 +2,99 @@
 //  LoginViewController.swift
 //  CourseGrab
 //
-//  Created by Daniel Vebman on 1/26/20.
+//  Created by Reade Plunkett on 1/26/20.
 //  Copyright © 2020 Cornell AppDev. All rights reserved.
 //
 
+import GoogleSignIn
+import SnapKit
 import UIKit
 
-class LoginViewController: UIViewController {
+class LoginViewController: UIViewController, GIDSignInDelegate {
+
+    private let stackView = UIStackView()
+    private let titleLabel = UILabel()
+    private let logoView = UIImageView()
+    private let signInButton = GIDSignInButton()
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
 
         view.backgroundColor = .white
+
+        GIDSignIn.sharedInstance().delegate = self
+        GIDSignIn.sharedInstance()?.presentingViewController = self
+        // Automatically sign in the user.
+        GIDSignIn.sharedInstance()?.restorePreviousSignIn()
+
+        setUpStackView()
```

Comment:
I think `setUp -> setup`

Reply:
We are using the verb "set up" not the noun so it should be camelCased since it's two different words